config/strategies/dauth.js

```diff
@@ -5,8 +5,8 @@ var https = require('https');
 
 function verify(token, callback) {
   var options = {
-    host: "login2.datasektionen.se",
-    path: "/verify/" + token + ".json?api_key=" + process.env.LOGIN2_KEY,
+    host: "login.datasektionen.se",
+    path: "/verify/" + token + ".json?api_key=" + process.env.LOGIN_KEY,
```

Comment:
This will require the modification of the variable's name in the container config before the patch is deployed. Keep that in mind! You can do this by using the `dokku config` command, but be aware that you will have to create a new key=value pair, since renaming is not an option. So copy the old key, save it in the new variable, and then delete the old one.

Reply:
Right, I'll make sure to change the config! :)